src/frontmenu_ingame_tabs.h

```diff
@@ -115,6 +115,23 @@ enum IngameButtonDesignationIDs {
     BID_OBJ_CLOSE,
     BID_OBJ_SCRL_UP, //80
     BID_OBJ_SCRL_DWN,
+    BID_POWER_TD17,
```

Comment:
Have you tested the new buttons with 'TUTURIAL FLASH BUTTON'?

Reply:
No. What do you mean?

Reply:
This command is there to highlight any interface button. Now you added new buttons, do they work along with this command still?

https://lubiki.keeperklan.com/dk1_docs/dk_scripting_ref.htm#scrpt_cmd_tutorial_flash_button

Reply:
TUTORIAL_FLASH_BUTTON(82, PLAYER0)

The Time Bomb icon flashes; however, the spell icon also flashes when I'm on the second spells panel.

Reply:
> TUTORIAL_FLASH_BUTTON(82, PLAYER0)
> 
> The Time Bomb icon flashes; however, the Spells Panel icon also flashes when I'm on the second spells panel.

That should be fixed now.